fe/fe-core/src/main/java/com/starrocks/analysis/CreateFunctionStmt.java

```diff
@@ -439,15 +443,27 @@ private void checkStarrocksJarUdafClass() throws AnalysisException {
             mainClass.checkArgumentCount(method, 1);
             mainClass.checkParamJavaType(method, udafStateClass.clazz, method.getParameters()[0]);
         }
+        if (isAnalyticFn) {
+            {
+                Method method = mainClass.getMethod(BATCH_UPDATE_METHOD_NAME, true);
```

Comment:
Why window function need batch update method？

Reply:
corresponds to our aggregation function method:
```
void update_batch_single_state(FunctionContext* ctx, AggDataPtr __restrict state, const Column** columns,
                                   int64_t peer_group_start, int64_t peer_group_end, int64_t frame_start,
                                   int64_t frame_end) const override
```